onyo/lib/git.py

```diff
@@ -450,24 +423,28 @@ def _parse_log_output(self, lines: list[str]) -> dict:
 
         return commit
 
-    def history(self, path: Path | None = None, n: int | None = None) -> Generator[dict, None, None]:
-        """Yields commit dicts representing the history of ``path``.
+    def history(self,
+                path: Path | None = None,
+                n: int | None = None) -> Generator[dict, None, None]:
+        """Yield commit dicts representing the history of ``path``.
 
-        History according to git log (git log --follow if a path is given).
+        The history is acquired via ``git log`` (``git log --follow`` if a
+        ``path`` is given).
 
         Parameters
         ----------
-        path:
-          What file to follow. If `None`, get the history of HEAD instead.
-        n:
-          Limit history going back ``n`` commits.
-          ``None`` for no limit (default).
+        path
+            The Path to get the history of. Defaults to ``HEAD`` (default).
```

Comment:
Now this sounds as if "HEAD" was a path like every other.

I know explaining this clearly is tricky, but I still want to highlight that I don't fully get it and feel like I have to guess.

Reply:
Yeah. I bumped on that too. Fixed.